plugins/jevents/jevents.php

```diff
@@ -65,7 +65,7 @@ function onContentBeforeSave($context, $data)
 			if ($result_count >= 1)
 			{
 				JFactory::getApplication()->enqueueMessage(JText::sprintf('JEV_CAT_MAN_DELETE_WITH_IDS',  $result_count), 'Warning');
-				JFactory::getApplication()->enqueueMessage(JText::_('JEV_CAT_DELETE_MSG_EVENTS_FIRST'), 'Warning');
+				JFactory::getApplication()->enqueueMessage(JText::sprintf('JEV_CAT_DELETE_MSG_EVENTS_FIRST'), 'Warning');
```

Comment:
You only use  JText::sprintf() when you have a variable to pass in to the format string.